packages/blocks/src/__internal__/utils/components.ts

```diff
@@ -33,6 +33,7 @@ export function BlockElement(
         <${model.tag}
           .model=${model}
           .host=${host}
+          class="affine-block-element"
```

Comment:
We have `getBoundingClientRect` to get the block element size which will be used for calculating the xywh prop for elements, however this API returns different values between block vs inline elements in Safari.
Making block elements to `display:block` fix this issue.
See
<img width="1146" alt="image" src="https://user-images.githubusercontent.com/584378/224893910-6d9bb670-8ac4-42c9-b51a-cddc02813089.png">
<img width="1130" alt="image" src="https://user-images.githubusercontent.com/584378/224894026-afb36b4d-8509-496c-9c89-b2fe0321999d.png">